.github/workflows/continuous-integration.yml

```
@@ -146,3 +146,37 @@ jobs:
 
       - name: Fetch Drupal cache
         run: yarn fetch-drupal-cache
+
+  build-e2e:
+    name: Build E2E
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get Node Version
```

Nit for consistency.

```suggestion
      - name: Get Node version
```